pkg/test/useraccount/assertion.go

```diff
@@ -51,10 +51,17 @@ func (a *Assertion) Exists() *Assertion {
 	return a
 }
 
+func (a *Assertion) HasEmbeddedSpec(spec toolchainv1alpha1.UserAccountSpecEmbedded) *Assertion {
```

Comment:
Well... UserAccount can't really have UserAccountSpecEmbedded (atm only MUR can have it). But it can match it... So I wonder if `MatchEmbeddedSpec()` would be a better name for that method. WDYT?
```suggestion
func (a *Assertion) MatchEmbeddedSpec(spec toolchainv1alpha1.UserAccountSpecEmbedded) *Assertion {
```

Reply:
@alexeykazakov agreed, that's a much better name. I'll update this!